scaling.go

```diff
@@ -99,6 +99,8 @@ func scaleWorkerThread(worker *worker) {
 	}
 
 	autoScaledThreads = append(autoScaledThreads, thread)
+
+	logger.LogAttrs(context.Background(), slog.LevelInfo, "upscaling worker thread", slog.String("worker", worker.name), slog.Int("threadIndex", thread.threadIndex), slog.Int("threads", len(autoScaledThreads)))
```

Comment:
```suggestion
	logger.LogAttrs(context.Background(), slog.LevelInfo, "upscaling worker thread", slog.String("worker", worker.name), slog.Int("thread", thread.threadIndex), slog.Int("num_threads", len(autoScaledThreads)))
```

For consistency. See #1699 and https://github.com/php/frankenphp/blob/main/frankenphp.go#L299.